database_scripts/propertySetup.py

```diff
@@ -160,10 +167,16 @@ def main():
          'year': '2023', 
          'columns_dict': {'PID': 'PID', 'ST_NUM': 'ST_NUM', 'ST_NAME': 'ST_NAME', 'UNIT_NUM': 'UNIT_NUM', 'CITY': 'CITY', 'ZIP_CODE': 'ZIP_CODE', 'OWNER': 'OWNER'}},
     ]
-    table_name = 'property'
 
     for data in api_data:
+        yr = data['year']
```

Comment:
So it looks like right now by default it's going to go through and import every year, which is obviously necessary the first time but is unnecessary on subsequent runs, and very time-consuming (as we discussed earlier). I'm trying to think of a way to modify this to better allow us to control going forward what years are imported. I see your comment at the top of the code that indicates that you should remove the years once they've been imported, but I'm not sure I like the idea of removing that code, I can see it being valuable to have it around (what if something happens to the existing database? or what if you just need to migrate to a new DB and re-import?). Another option is to have a static list defined in the code controlling what years are imported. One more option is a command line flag. Then we could add an if statement on line 173 that ensures that json_to_table is only called if the 'yr' variable is included in that list of years. What do you think?

Reply:
I think that way will be easier for other people to use the script later one. Fixing them right now.